starter-code/src/functions-and-arrays.js

```diff
@@ -1,6 +1,19 @@
 // Find the maximum
+function maxOfTwoNumbers (num1, num2) {
+  return num1 > num2 ? num1 : num2;
+}
 
 // Finding Longest Word
+function findLongestWord(wordArray) {
+  var longestWord = wordArray[0];
+
+  for (var i = 1; i < wordArray.length; i++) 
+    if (longestWord.length < wordArray[i].length)
+      longestWord = wordArray[i];  
+
+  return longestWord;
+}
```

Comment:
Bien hecho, recordad, se puede no poner las llaves de apertura y cierre en javascript si el bloque de código solo alcanza una línea, pero es más legible si las ponéis. Mirad, después de encontrar la palabra más larga manualmente, el método filter( ) y aplicadlo a este ejercicio